docs/releases/3.1.txt

```diff
@@ -496,6 +496,34 @@ Miscellaneous
 Features deprecated in 3.1
 ==========================
 
+Logout via GET
+--------------
+* Logging out via ``GET`` requests to the
+  :py:class:`built-in logout view <django.contrib.auth.views.LogoutView>` is
+  deprecated. Please update your code to use ``POST`` instead.
+
+  If you want to retain the user experience of an HTML link, you can use a
+  form that is styled to appear as a link:
+
+.. code-block:: html
+
+  <form id="logout-form" method="post" action="{% url 'admin:logout' %}">
+    {% csrf_token %}
+    <button type="submit">{% translate "Log out" %}</button>
+  </form>
+
+.. code-block:: css
+
+  #logout-form {
+    display: inline;
+  }
+  #logout-form button {
+    background: none;
+    border: none;
+    cursor: pointer;
+    padding: 0;
+  }
```

Comment:
@django/accessibility Can you check this suggestion? Are there any accessibility issue with this snippet? :thinking:

Reply:
Looks okay to me. Ideally it should look like a button but if they're wanting it to look like a link, that's probably the best way.

Reply:
Thanks :+1:

Reply:
Thanks for checking! We’d need an additional `text-decoration: underline;` so this appears like a link – otherwise there’s no visual cue this is clickable.

```suggestion
  <form id="logout-form" method="post" action="{% url 'admin:logout' %}">
    {% csrf_token %}
    <button type="submit">{% translate "Log out" %}</button>
  </form>

.. code-block:: css

  #logout-form {
    display: inline;
  }
  #logout-form button {
    background: none;
    border: none;
    cursor: pointer;
    padding: 0;
    text-decoration: underline;
  }
```

Reply:
Thanks :+1: